.circleci/config.yml

```diff
@@ -39,6 +39,14 @@ jobs:
                 - universal-login-{{ .Environment.CIRCLE_SHA1 }}
       - run: cd universal-login-sdk && yarn test:nobuild
       - run: cd universal-login-relayer && yarn test:nobuild
+  test_react:
+    docker:
+      - image: circleci/node:10.15
+    steps:
+      - restore_cache:
+            keys:
+                - universal-login-{{ .Environment.CIRCLE_SHA1 }}
+      - run: cd universal-login-react && yarn test
   test_wallet_example:
```

Comment:
merge test_react and test_wallet_example

Reply:
👍